test/language/statements/async-function/evaluation-this-value-global.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2016 Microsoft, Inc. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Brian Terlson <brian.terlson@microsoft.com>
+es7id: pending
+description: >
+  The this value is set to the global object when not passed in sloppy mode.
+flags: [noStrict]
+---*/
+
+var glob = Function("return this;")();
```

Comment:
Actually, scratch that. Since gh-568 and gh-595, we are now encouraging the use of the `this` keyword in the global scope to access the global object.